Nimble Templates/iOS Sources.xctemplate/TemplateInfo.plist

```diff
@@ -0,0 +1,158 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+    <key>Kind</key>
+    <string>Xcode.Xcode3.ProjectTemplateUnitKind</string>
+    <key>Identifier</key>
+    <string>co.nimble.ios-template.sources</string>
+
+	<key>Definitions</key>
+    <dict>
+        <key>Application/AppDelegate/temp</key>
+        <dict>
+			<key>Group</key>
```

Comment:
@blyscuit Sorry to bother you again, but please check again about the indent on this file, otherwise everything is good to me

Reply:
I think I have problem with the indent setting in VSCode 😓 Will try to get it working.

Reply:
The indent should be correct but when previewed in Github, the setting for indent changed. Any advice I'm not familiar with this error.

Reply:
I added .editorconfig. Should solve the issue with indent preview.